crypto/testutil.go

```diff
@@ -0,0 +1,29 @@
+package crypto
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+)
+
+// StaticVRF returns a static VRF private key for _tests_.
+func StaticVRF(t *testing.T) vrf.PrivateKey {
+	sk, err := vrf.GenerateKey(bytes.NewReader(
+		[]byte("deterministic tests need 256 bit")))
+	if err != nil {
+		t.Fatal(err)
+	}
+	return sk
+}
+
+// StaticSigning returns a static private signing key for _tests_.
+func StaticSigning(t *testing.T) sign.PrivateKey {
```

Comment:
I think we should use consistent naming and include the word `Test` in these function names just to make it absolutely clear to the reader that these are used for testing.

Reply:
I still prefer to keep this name since it reflects exactly what the function does. Besides, the docstring states it clearly and the parameter requires `testing.T` so I don't think this will cause any trouble.

Reply:
My point was a bit more about consistency in naming throughout all of our tests, we should agree on a convention. You're definitely right, if the docstring states the test clearly, the name doesn't need to include `Test`.

Reply:
`Test` is used as prefix only for testcases in `_test.go` files, so I think this doesn't violate our convention ;)

Reply:
This is true for the test cases. At the same time, for this function I would have expected a name more like `NewTestSigningKey` since most other similar `testutil` helpers follow the `NewTest* convention.

Reply:
What do you think about https://github.com/coniks-sys/coniks-go/pull/191/commits/e06784bb925b4e2b560af6f80a77dcf3db9a4e83?

Reply:
This definitely works. Thanks!